### ISSUE TITLE
Error when compiling taint2 plugin

### ISSUE THREAD

**Issue by blazef104** [panda-re/panda#804]
Hello, when trying to compile the taint2 plugin with the build script: `./build.sh i386-softmmu --disable-werror --disable-pyperipheral3` I get the following error.
```# C protobuf
  PROTO   i386-softmmu/plog.pb-c.h i386-softmmu/plog.pb-c.c 
plog.proto:132:21: "ptr" is already defined in "panda.TaintQueryUniqueLabelSet".
plog.proto:133:21: "label" is already defined in "panda.TaintQueryUniqueLabelSet".
plog.proto:131:9: "TaintQueryUniqueLabelSet" is already defined in "panda".
plog.proto:136:21: "filename" is already defined in "panda.SrcInfo".
plog.proto:137:21: "linenum" is already defined in "panda.SrcInfo".
plog.proto:138:21: "astnodename" is already defined in "panda.SrcInfo".
plog.proto:139:21: "insertionpoint" is already defined in "panda.SrcInfo".
plog.proto:140:21: "ast_loc_id" is already defined in "panda.SrcInfo".
plog.proto:135:9: "SrcInfo" is already defined in "panda".
plog.proto:143:21: "ptr" is already defined in "panda.TaintQuery".
plog.proto:144:21: "tcn" is already defined in "panda.TaintQuery".
plog.proto:145:21: "offset" is already defined in "panda.TaintQuery".
plog.proto:146:39: "unique_label_set" is already defined in "panda.TaintQuery".
plog.proto:142:9: "TaintQuery" is already defined in "panda".
plog.proto:149:21: "info" is already defined in "panda.AttackPoint".
plog.proto:150:24: "call_stack" is already defined in "panda.AttackPoint".
plog.proto:151:22: "src_info" is already defined in "panda.AttackPoint".
plog.proto:148:9: "AttackPoint" is already defined in "panda".
plog.proto:154:21: "buf" is already defined in "panda.TaintQueryHypercall".
plog.proto:155:21: "len" is already defined in "panda.TaintQueryHypercall".
plog.proto:156:21: "data" is already defined in "panda.TaintQueryHypercall".
plog.proto:157:21: "num_tainted" is already defined in "panda.TaintQueryHypercall".
plog.proto:158:24: "call_stack" is already defined in "panda.TaintQueryHypercall".
plog.proto:159:22: "src_info" is already defined in "panda.TaintQueryHypercall".
plog.proto:160:25: "taint_query" is already defined in "panda.TaintQueryHypercall".
plog.proto:153:9: "TaintQueryHypercall" is already defined in "panda".
plog.proto:211:30: "taint_query_hypercall" is already defined in "panda.LogEntry".
plog.proto:212:22: "attack_point" is already defined in "panda.LogEntry".
plog.proto:213:17: "taint_label_virtual_addr" is already defined in "panda.LogEntry".
plog.proto:214:17: "taint_label_physical_addr" is already defined in "panda.LogEntry".
plog.proto:215:17: "taint_label_number" is already defined in "panda.LogEntry".
plog.proto:211:54: Field number 38 has already been used in "panda.LogEntry" by field "taint_query_hypercall".
plog.proto:212:37: Field number 39 has already been used in "panda.LogEntry" by field "attack_point".
plog.proto:213:44: Field number 6 has already been used in "panda.LogEntry" by field "taint_label_virtual_addr".
plog.proto:214:45: Field number 7 has already been used in "panda.LogEntry" by field "taint_label_physical_addr".
plog.proto:215:38: Field number 8 has already been used in "panda.LogEntry" by field "taint_label_number".
/home/giulio/panda/panda/Makefile.panda.target:77: recipe for target 'plog.pb-c.h' failed
make[1]: *** [plog.pb-c.h] Error 1
Makefile:357: recipe for target 'subdir-i386-softmmu' failed
make: *** [subdir-i386-softmmu] Error 2
```
I'm on ubuntu 18.04 with llvm3 installed and protobuf3, it's a clean installation of ubuntu and all the dependencies were installed using the install_ubuntu.sh script. Does anyone know what is causing those errors?

**Comment by blazef104** [comment]
I just resolved the issue.
tl;dr: just remove the taint2 plugin from the config.panda

Until now it was not clear to me that I could write a plug-in using functions from another without having to compile this list one. I thought that in order to have a let’s say “taint_string” plug-in one had to compile both the taint2 and taint_string plugin. However the only way I got this to work is by removing taint2 from the list of plugins to be compiled and panda will take care of the rest. 

This issue can be closed, hope it can be helpful to someone else that just started to use panda and is getting familiar with how it works :)
